backend/src/routes/api.ts

```diff
@@ -27,25 +27,30 @@ router.get("/baskets", async (_req: Request, res: Response) => {
   }
 });
 
-
 router.get("/generate_name", async (_req: Request, res: Response) => {
   let basketName: string = "";
 
   do {
-    basketName = generate_random_string().substring(2, 9);
+    basketName = generateRandomString().substring(2, 9);
   } while ((await isBasketNameUnique(basketName)) === false);
 
   res.status(200).json({ basketName });
 });
 
-router.get("/generate_token", async (_req: Request, res: Response) => {
-  let token: string = await generate_token();
-  await storeToken(token);
+router.get("/generate_token", async (req: Request, res: Response) => {
+  const basketName = req.query.name;
```

Comment:
This endpoint doesn't require a basket name at the moment. Should it include it?